sklearn/svm/classes.py

```diff
@@ -446,20 +446,29 @@ class SVC(BaseSVC):
         Penalty parameter C of the error term.
 
     kernel : string, optional (default='rbf')
-         Specifies the kernel type to be used in the algorithm.
-         It must be one of 'linear', 'poly', 'rbf', 'sigmoid', 'precomputed' or
-         a callable.
-         If none is given, 'rbf' will be used. If a callable is given it is
-         used to pre-compute the kernel matrix from data matrices; that matrix
-         should be an array of shape ``(n_samples, n_samples)``.
+        Specifies the kernel type to be used in the algorithm.
+        It must be one of 'linear', 'poly', 'rbf', 'sigmoid', 'precomputed' or
+        a callable.
+        If none is given, 'rbf' will be used. If a callable is given it is
+        used to pre-compute the kernel matrix from data matrices; that matrix
+        should be an array of shape ``(n_samples, n_samples)``.
 
     degree : int, optional (default=3)
         Degree of the polynomial kernel function ('poly').
         Ignored by all other kernels.
 
     gamma : float, optional (default='auto')
         Kernel coefficient for 'rbf', 'poly' and 'sigmoid'.
-        If gamma is 'auto' then 1/n_features will be used instead.
+
+        Current default is 'auto' which uses 1 / n_features,
+        if ``gamma='scale'`` is passed then it uses 1 / (n_features * X.std())
+        as value of gamma. The current default of gamma, 'auto', will change
+        to 'scale' in version 0.22. 'auto_deprecated', a deprecated version of
+        'auto' is used as a default indicating that no explicit value of gamma
+        was passed.
+
+        .. versionchanged:: 0.20
```

Comment:
Not quite sure about what to put as versionchanged here. A bit confusing at first but I guess "versionchanged:: 0.20" is fine ...

Reply:
Still unsure about this one. Either we:
* do not add anything now and try to remember to add a versionchanged in 0.22 to say that the default has changed. 
* use `versionchanged:: 0.22` and write it as a message from our future selves i.e. "default has changed in 0.22 from 'auto' to 'scale'.

I think I favour option 1. less confusing.

Reply:
Sorry I somehow missed the comment here. I'm fine with removing versionchanged
(1) There's enough information in parameter description
(2) The change will actually occurs in 0.22, not 0.20.